net.lutris.Lutris.yml

```diff
@@ -14,9 +14,11 @@ finish-args:
   - --share=network
   - --allow=multiarch
   - --allow=devel
+  - --talk-name=org.freedesktop.Flatpak.*
```

Comment:
Why it needs this line? Its for the wrapper? This is a somewhat dangerous hole in the sandbox.

Reply:
One alternative is to not add this hole and explicitly say how-to in the README.

Reply:
> Why it needs this line? Its for the wrapper? This is a somewhat dangerous hole in the sandbox.

Appimages don't start without it and every other flatpak implementing fuse has this line.
Sadly I couldn't find out why exactly it needs this line.

> One alternative is to not add this hole and explicitly say how-to in the README.

Yes, that might be a good idea.
I'm also not completely comfortable with how much it compromises the sandbox.

Reply:
Yeah, lets do that, that particular hole allows for arbitrary code execution.